### DISCUSSION TITLE
Website bug report

### DISCUSSION THREAD

**Discussion by Semigradsky** [vercel/next.js#23050] [Help] [answered]
If found bug in site (https://nextjs.org/). How I can report it?

There is no "Website bug report" in list https://github.com/vercel/next.js/issues/new/choose

**Comment by ErnestBrandi** [comment, marked as answer]
_Maybe_ go on Vercel.com, there is a "Support" button in the top right part of the navbar.

**Reply by leerob** [reply]
Yeah, that would work! Thank you @Semigradsky 👍
